### DISCUSSION TITLE
Merge peaks from a subset of cell types using iterative overlap

### DISCUSSION THREAD

**Discussion by Dazcam** [GreenleafLab/ArchR#2007] [Questions / Documentation]
Hi there,

I have run the `ArchR` pipeline up until the peak calling stage on a dataset derived from brain tissue and I'm looking to generate a merged peak set similar to the union peak set generated for all cell types, but only for a subset of the cell types (i.e. the neuronal cell types). In effect, I'm looking to merge the cell specific neuronal peak sets (saved in the`PeakCalls` directory). I would normally use something like bedtools or bedops for this but would like to use the iterative overlap method described in the `ArchR` docs if possible.

Note that I don't want to replace the original union peak set stored in the `ArchR` object, just to create a new peak set in the `PeakCalls` directory for all peaks detected in neurones. 

Is there a function / method available to do this within `ArchR`?

Many Thanks.

**Comment by rcorces** [comment]
Create a new column in `cellColData` with `NA` for cells you dont want included and the rest of your neuron groups defined. Then `addReproduciblePeakSet()` on that column. This will replace the peak set that is associated with your project but I think it should maintain the file corresponding to the old peak set in your project directory.
